controllers/keda/transform/transform.go

```diff
@@ -391,6 +503,10 @@ func EnsureAuditPolicyConfigMapMountsVolume(configMapName string, scheme *runtim
 	}
 }
 
+// func ReplaceKedaOperatorCertRotation(enabled bool, scheme *runtime.Scheme, logger logr.Logger) mf.Transformer {
```

Comment:
Did you mean to leave this here?

Reply:
good catch, a leftover from a previous version